docs/versioned_docs/version-zenith/user/quickstart/183917-daggerverse.mdx

```diff
@@ -0,0 +1,33 @@
+---
+slug: /user/quickstart/183917/daggerverse
+hide_table_of_contents: true
+title: "Use the Daggerverse"
+---
+import PartialExperimentalDocs from '../../partials/_experimental.mdx';
+
+# Quickstart
+
+## Use the Daggerverse
+
+<PartialExperimentalDocs />
+
+Modules don't need to be installed locally. The [Daggerverse](https://daggerverse.dev) is an online catalog of Dagger modules that enables users to easily discover and use modules created by the Dagger community.
+
+You can `dagger call` a function from a module using its Daggerverse URL. Here's an example of using a module from the Daggerverse to perform a Trivy security scan of an image:
+
+```shell
+dagger call -m github.com/jpadams/daggerverse/trivy scan-image --image-ref alpine/git:latest
+```
+
+All Dagger CLI commands work as expected with Daggerverse URLs. Here are a few more examples for you to try:
+
+- `dagger call -m github.com/shykes/daggerverse/ttlsh publish --ctr alpine:latest`
+  - Module information: https://daggerverse.dev/mod/github.com/shykes/daggerverse/ttlsh@16e40ec244966e55e36a13cb6e1ff8023e1e1473
+- `dagger -m github.com/shykes/daggerverse/wolfi shell base with-package --name curl with-package --name cowsay container`
+   - Module information: https://daggerverse.dev/mod/github.com/shykes/daggerverse/wolfi@faf1103e0b125477d840d2b96a937518ebc0d5bd
+- `dagger -m github.com/sipsma/daggerverse/yamlinvaders shell play`
+   - Module information: https://daggerverse.dev/mod/github.com/sipsma/daggerverse/yamlinvaders@bbdd8736c41ab7d867a62e3db32895c143e78248
+- `dagger up --native --mod github.com/helderco/daggerverse/python@kcna23 http-server --src <path/to/web/assets>`
+  - Module information: https://github.com/helderco/daggerverse/tree/main/python/kcna23
```

Comment:
This is a bit hard to read, could turn the module information as a simple link with the module's name, e.g., [ttlsh](https://daggerverse.dev/mod/github.com/shykes/daggerverse/ttlsh@16e40ec244966e55e36a13cb6e1ff8023e1e1473). Maybe a short description of what the function does:

Reply:
Also, the command may not correspond with the "more information" URL since at the time the user makes the call, it may have a different hash.

Reply:
Modified